pmlc/target/x86/pipeline.cc

```diff
@@ -262,7 +262,10 @@ void pipelineBuilderStage1(OpPassManager &pm) {
   }
 
   pm.addPass(pmlc::conversion::tile_to_linalg::createLowerTileToLinalgPass());
-  pm.addNestedPass<FuncOp>(createReorderLayoutsPass());
+  if (!util::getEnvVar("PLAIDML_REORDER_WEIGHTS").empty())
```

Comment:
Do I get it correctly after review the changes, that the default is the re-order weights and activation (and according to the td file try to do with a channel blocking of 32) and we set a env var to just reorder weights?

Reply:
Yes, so that I can more easily compare perf. Once we confirm perf is better for activations + weights we can drop the reorder weights-only pass.